Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'pry'
 # If you have any plugins, put them here!
 group :jekyll_plugins do
    gem "jekyll-feed", "~> 0.11"
-   gem "jekyll-multiple-languages-plugin"
+   gem "jekyll-multiple-languages-plugin", github: "kurtsson/jekyll-multiple-languages-plugin", ref: "refs/pull/178/head"
```

Comment:
This is necessary, to exclude the English language version of `/brasil` which does not exist, as only `/pt/brasil` exists. When the gem version is updated, we can point to that.